cross/bitlbee/Makefile

```diff
@@ -11,6 +11,8 @@ HOMEPAGE = http://www.bitlbee.org/
 COMMENT  = BitlBee brings IM (instant messaging) to IRC clients. It\'s a great solution for people who have an IRC client running all the time and don\'t want to run an additional MSN/AIM/whatever client
 LICENSE  =
 
+include ../../mk/spksrc.common.mk
```

Comment:
hmmm. I think using **cross-cc.mk** before our variables at least for **TARGET_PATCH, TARGET_INSTALL, etc**.
That's why I applied added **common.mk** before the variable declarations used in **cross-cc.mk**.
I have to redo all changes again:
>LICENSE...
>
>GNU_CONFIGURE ...if applicable
>
>include ../../mk/spksrc.common.mk
>
>CONFIGURE_ARGS = ...if applicable
>
>ifeq($(findstring $(ARCH)...
>  <thing>
>endif
>
>include ../../mk/spksrc.cross-cc.mk
>
>.PHONY etc. ...if applicable
>